lti_auth/templates/lti_auth/fail_course_configuration.html

```diff
@@ -14,7 +14,7 @@
                     <small>
                         Please contact your help desk to enable the component.<br />
 
-                        {% if 'administrator' in roles or 'instructor' in roles %}
+                        {% if is_instructor or is_administrator %}
                             <h3>Administrators & Instructors</h3>
```

Comment:
@sdreher do you need an html entity here?

Reply:
good catch. yes I do.